project/Settings.scala

```diff
@@ -41,7 +41,7 @@ object Settings {
     "-language:implicitConversions",     // Allow definition of implicit functions called views
     "-unchecked",                        // Enable additional warnings where generated code depends on assumptions.
     "-Xcheckinit",                       // Wrap field accessors to throw an exception on uninitialized access.
-    "-Xfatal-warnings",                  // Fail the compilation if there are any warnings.
+    //"-Xfatal-warnings",                  // Fail the compilation if there are any warnings.
```

Comment:
🫣 - curious what warnings you were seeing?

Reply:
we should bring this back...otherwise I can see the repo starts to get more and more unused imports :)